dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -37,18 +37,19 @@ public class StepwisePlanner
     public StepwisePlanner(
         IKernel kernel,
         StepwisePlannerConfig? config = null,
-        string? prompt = null)
+        string? prompt = null,
+        PromptTemplateConfig? promptUserConfig = null)
     {
         Verify.NotNull(kernel);
         this._kernel = kernel;
 
         this.Config = config ?? new();
         this.Config.ExcludedSkills.Add(RestrictedSkillName);
 
-        var promptConfig = new PromptTemplateConfig();
+        var promptConfig = promptUserConfig ?? new PromptTemplateConfig();
         var promptTemplate = prompt ?? EmbeddedResource.Read("Skills.StepwiseStep.skprompt.txt");
         string promptConfigString = EmbeddedResource.Read("Skills.StepwiseStep.config.json");
-        if (!string.IsNullOrEmpty(promptConfigString))
+        if (!string.IsNullOrEmpty(promptConfigString) && promptUserConfig != null)
```

Comment:
` && promptUserConfig != null`
I think this is inverse of what we'd want, no? If promptUserConfig *IS* null, then we set promptConfig to the value loaded from file.

Reply:
My bad. Fixed